articles/link-accounts/index.md

```diff
@@ -330,3 +330,31 @@ The result of the unlinking process is the following:
 * The secondary account will have no metadata
 
 If your goal is to delete the secondary identity entirely, you must first unlink the accounts, and then delete the newly created secondary account.
+
+## Properties from secondary identities
+
+When a user logs in, applications receive user information from the **primary identity**. Auth0 will not attempt to automatically complete any missing profile field with information from the secondary identities. For example, if the primary identity comes from a database connection and is missing the `given_name` and `family_name` properties, and the secondary identity comes from a Google social connection that includes the first and last name of the user, then by default, the application will **not** receive data contained in the second identity.
+
+To fill in the missing info in primary identities with information from secondary identities, you can write a [rule](/rules):
+
+```js
+function(user, context, callback) {
+  
+  const propertiesToComplete = ["given_name", "family_name", "name", "maroon"];
```

Comment:
What is the "maroon" property in this list? It isn't immediately apparent to a new customer. It looks like the name of a color, but we've been discussing names this whole time?

Reply:
My bad... It was there from a local test. Just removed it. Thanks!